lang/cs/Org.Apache.REEF.Client/API/JobRequestBuilder.cs

```diff
@@ -204,5 +244,23 @@ public JobRequestBuilder SetJavaLogLevel(JavaLoggingSetting javaLogLevel)
             _jobParametersBuilder.SetJavaLogLevel(javaLogLevel);
             return this;
         }
+
+        /// <summary>
+        /// Returns true, if the given file path references a DLL or EXE or JAR.
+        /// </summary>
+        /// <param name="filePath"></param>
+        /// <returns></returns>
+        private static bool IsAssemblyToCopy(string filePath)
```

Comment:
Where is this used?

Reply:
`AddGlobalAssemblies` -- this was the previous behavior to only add in JAR, EXE, and DLL from a specified directory.